src/assets/scss/_cards.scss

```diff
@@ -93,28 +98,112 @@
 
 }
 
-.card-tile {
+.module-tile {
```

Comment:
Restructured this so pretty much any element that has a hover state (with your fancy animation) is defined one time. Added a few variants (ex: the Top Story image, articles within groups, and social posts).

Reply:
Also renamed this so its mentally harder to get card TILE mixed up with TITLES.